rulebooks/dnd5e/character/draft.go

```diff
@@ -818,6 +837,36 @@ func (d *Draft) compileFeatures() ([]features.Feature, error) {
 	return featureList, nil
 }
 
+// compileConditions creates conditions from draft choices (e.g., fighting styles)
+func (d *Draft) compileConditions(characterID string) ([]dnd5eEvents.ConditionBehavior, error) {
+	conditionList := make([]dnd5eEvents.ConditionBehavior, 0)
+
+	// Look for fighting style choices
+	for _, choice := range d.choices {
+		if choice.Category == shared.ChoiceFightingStyle && choice.FightingStyleSelection != nil {
```

Comment:
at min, this should be abstarcted away. is this choice valid? ideally we should only have valid choices we iterate over